app/services/get_from_doit.rb

```diff
@@ -0,0 +1,30 @@
+class GetFromDoit
+
+	def self.run
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for run is too high. [28.91/15] (http://c2.com/cgi/wiki?AbcMetric)<br>Metrics/MethodLength: Method has too many lines. [24/10] (https://github.com/bbatsov/ruby-style-guide#short-methods)